applications/conceptual-model-editor/src/action/actions-react-binding.tsx

```diff
@@ -57,6 +66,8 @@ interface DialogActions {
 
 interface VisualModelActions {
 
+  addSemanticEntityToVisualModel: (identifier: string, position?: { x: number, y: number } | null) => void;
```

Comment:
You are right for consistency it should be `null` or `given pos`. The reason why I put the `undefined` here was that I was looking at it as kind of lazy method, because you can always call the right method based on the type of entity, if you are not lazy to check. And usually you don't even have to check when you know the type of entity right away.

You also mention it further down that you already removed the method once.

Reply:
Also for some of the entities (like relationships) the position doesn't do anything, so that was one more reason.